# HG changeset patch
# User Nathan Phillip Brink <ohnobinki@ohnopublishing.net>
# Date 1320859299 0
# Node ID 110ba58ecd56ad5debb00726e5e3ca8d5df135bf
# Parent  a673dd0f7b377ec35ce26fffc4affc9c7563c7b1
- Fix compilation issue when disabling stacked extbans. https://bugs.gentoo.org/389949

diff -r a673dd0f7b37 -r 110ba58ecd56 Changes
diff -r a673dd0f7b37 -r 110ba58ecd56 src/extbans.c
--- a/src/extbans.c	Wed Nov 09 17:16:55 2011 +0000
+++ b/src/extbans.c	Wed Nov 09 17:21:39 2011 +0000
@@ -533,9 +533,9 @@
 	req.conv_param = extban_conv_param_nuh;
 #else
 	req.conv_param = extban_conv_param_nuh_or_extban;
+	req.is_ok = extban_is_ok_nuh_extban;
 #endif
 	req.is_banned = extban_modej_is_banned;
-	req.is_ok = extban_is_ok_nuh_extban;
 	req.options = EXTBOPT_ACTMODIFIER;
 	ExtbanAdd(NULL, req);
 
